code/modules/mob/dead/observer/observer.dm

```diff
@@ -249,7 +249,7 @@
 
 ///makes the ghost see the target hud and sets the eye at the target.
 /mob/dead/observer/proc/do_observe(mob/target)
```

Comment:
```suggestion
/mob/dead/observer/proc/do_observe(atom/movable/target)
```

Reply:
Will do. Needed to move the `var/mob/living/carbon/human/human_target` cast a bit earlier and assign more thoroughly though. Also `ismovableatom()` can stay its orginal `istype()`.